plugins/modules/mysql_replication.py

```diff
@@ -381,43 +381,35 @@ def changemaster(cursor, chm, connection_name='', channel=''):
 
 
 def main():
+    argument_spec = mysql_common_argument_spec()
+    argument_spec.update(
+        mode=dict(type='str', default='getslave', choices=[
+            'getmaster', 'getslave', 'changemaster', 'stopslave',
+            'startslave', 'resetmaster', 'resetslave', 'resetslaveall']),
+        master_auto_position=dict(type='bool', default=False),
```

Comment:
Same question here, why removing some of the arguments ?

Reply:
Same as above.